File: media/forms/EXM060/frmEXM060.py
from django.shortcuts import render, HttpResponse
import json

def load(request):
    return render(request,'minimal-frame.html',{'widget':'forms/EXM060/frmEXM060'})

def word (request):
    params = json.loads(request.body)
    text1 = params['txtTest1']['value']
    text2 = params['txtTest2']['value']
    result = {'result' : text1.count(text2)}
    return HttpResponse(json.dumps(result))

File: media/forms/EXM060/frmEXM060.js
define([
    "dojo/_base/declare",
    "common/MinimalForm",
    "dojo/text!./frmEXM060.html",
    "dojo/on",
], function(declare, MinimalForm, template, on) {

    return declare([MinimalForm], {
        metag: 'EXM060',
        baseURL: 'EXM060',
        templateString: template,

        startup: function() {
            this.inherited(arguments);
            var widget = this;
            on(widget.cmdTest1, 'click', function(){
                
                var data = {};
                widget.param(data,'txtTest1',['value']);
                widget.param(data,'txtTest2',['value']);
                widget.post('word',data,function(result){
                    widget.txtTest3.set('value',result.result);

                });
            });
        },

    });
});
